ledger/ledger-api-tests/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Reporter.scala

```diff
@@ -58,7 +58,8 @@ object Reporter {
 
     private def indented(msg: String, n: Int = 2): String = {
       val indent = " " * n
-      msg.linesIterator.map(l => s"$indent$l").mkString("\n")
+      if (msg != null) msg.linesIterator.map(l => s"$indent$l").mkString("\n")
+      else ""
```

Comment:
Or with an `Option(msg)` if you want to avoid the 2-lines branch.

Reply:
I'm gonna stick to the explicit version, thanks.